internal/controllers/reconciler.go

```diff
@@ -198,6 +204,13 @@ func (r *Reconciler) reconcile(ctx context.Context, cr *model.CryostatInstance)
 
 	// Set up TLS using cert-manager, if available
 	var tlsConfig *resources.TLSConfig
+
+	// Reconcile RBAC resources for Cryostat
+	err = r.reconcileRBAC(ctx, cr)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
```

Comment:
Just a small nit, could I bug you to move this block above the `// Set up TLS using cert-manager, if available` comment? That refers to the code after `reconcileRBAC`.